src/rpm/signature/pgp.rs

```diff
@@ -336,22 +341,21 @@ pub(crate) mod test {
         let passwd_fn = || String::new();
 
         let now = now();
-        let mut sig_cfg_bldr = ::pgp::packet::SignatureConfigBuilder::default();
-        let sig_cfg = sig_cfg_bldr
-            .version(::pgp::packet::SignatureVersion::V4)
-            .typ(::pgp::packet::SignatureType::Binary)
-            .pub_alg(::pgp::crypto::public_key::PublicKeyAlgorithm::RSA)
-            .hash_alg(::pgp::crypto::hash::HashAlgorithm::SHA2_256)
-            .issuer(Some(signer.secret_key.key_id()))
-            .created(Some(now))
-            .unhashed_subpackets(vec![]) // must be initialized
-            .hashed_subpackets(vec![
-                ::pgp::packet::Subpacket::SignatureCreationTime(now),
-                ::pgp::packet::Subpacket::Issuer(signer.secret_key.key_id()),
+
```

Comment:
The only downside I see here is the kids of sanitization checks `.build()` would do assuming it does some.